src/main/webapp/feed.html

```diff
@@ -23,24 +23,48 @@ <h1>Message Feed</h1>
 
     function MessageFeed() {
       const [allMessages, setMessages] = React.useState([])
```

Comment:
Please use semicolons; see https://google.github.io/styleguide/jsguide.html#formatting-semicolons-are-required.

Automatic semicolon insertion can produce unexpected parsing in some cases.